### DISCUSSION TITLE
How did y’all discover toonsquid?

### DISCUSSION THREAD

**Discussion by Cjjuntti** [keiwando/toonsquid#612] [General]
So I’ve used toonsquid for over a year now and it’s probably one of the best animation apps I’ve ever used. I first discovered toonsquid because of either brad colbow or kittydog? Either way, this was my early introduction to the app and since then, it’s become my go to for animation. I’d like to know how y’all found out about toonsquid, and how would you get it out that it’s the best animation app on iPad?

**Comment by DWrightAnimation** [comment]
I discovered the app through Brad Colbow's video on it. And if you ask me, I'd get it out there that it's the best mobile animation app of all time by posting about it everywhere I see fit.

**Comment by trinhpxl** [comment]
searching youtube, “new animation app” filtered by month xD ! i found toonsquid 15 min basics there,, a month after it was uploaded that i found it, i think ? anyways, after a few weeks i started advertising it on [tiktok](https://www.tiktok.com/t/ZPREujkoq/) ! 

people started asking questions about the app so i started answering. i actually told a popular animation memer who goes by luckiziy in their discord about toonsquid and then they switched to it too ! (they still mostly use flipaclip but do bring awareness to TS in their vid titles) so that helped spread word about TS, kittydog started using it, other animation memers started using it too, some of people i asked who used toonsquid said they saw me on tiktok too lmao 😭😭

then someone opfrom youtube decided to start a [toonsquid discord server,](https://discord.com/invite/83C3qcmPmQ) they needed help setting stuff up so i decided to become a mod for that server ! we run monthly contests now with money prizes !

**Reply by budaloops** [reply]
I vaguely remember hearing someone talking about it in a video. Or a Reddit post when it was fairly new. Someone was saying how feature-rich it is for a low price and that sounded exciting.

I would search for content on creative iPad apps often

**Comment by DinoAwakening** [comment]
The way I found out about ToonSquid might sound boring (and it is)… There was I, tired from the good old RoughAnimator app, being the most capable animation app on mobile at that moment (or that’s what I thought) until I scrolled really deep down in the App Store searching by the name “Callipeg”. And since I didn’t liked Callipeg itself that much I was expecting to find something such as capable or even better. At first I saw a “Weird looking rainbow squid”, and doubt about buying it cause there wasn’t many people talking about the app or downloading it even tho it looked great. But, well, I can’t help myself since I buy almost anything that catches my attention and here am I… With you guys xD

**Comment by ll-ghin** [comment]
Procreate Dreams absolutely sucked and I was so disappointed so I went searching for alternatives and found Toonsquid and now I exclusively animate in Toonsquid.  I tried Toonboom Harmony, TVPaint, Adobe Animate but Toonsquid's convenience on iPad is just too great.

**Comment by ShnitzelKiller** [comment]
I saw a thread on Procreate discussion forum which is a wasteland of disappointed buyers of Procreate Dreams, pointing out all the missing features in Dreams that Toonsquid has.

**Comment by Gottafindaname** [comment]
I discovered it though the ipad animator discord server. Now i only use toonsquid
